src/stockwatch/app/runtime.py

```diff
@@ -1,9 +1,65 @@
 """Runtime data used for displaying the Stockwatch site."""
-from stockwatch.use_cases import degiro
+from datetime import date
 
-_SCRAPE_THREAD = degiro.ScrapeThread()
+from stockwatch import entities, use_cases
 
+_SCRAPE_THREAD = use_cases.degiro.ScrapeThread()
+_PORTOS: tuple[entities.shares.SharePortfolio, ...] = ()
+_INDICES: list[tuple[entities.shares.SharePosition, ...]] = []
 
-def get_scrape_thread() -> degiro.ScrapeThread:
+
+def load_portfolios() -> None:
+    """Load portfolios from the stockwatch dir"""
+    global _PORTOS  # pylint: disable=global-statement
+    global _INDICES  # pylint: disable=global-statement
+    _PORTOS, _INDICES = use_cases.get_portfolios_index_positions()
+
+
+def get_portfolios(
```

Comment:
The name is confusing, almost the same as in use_cases. I propose something like "portfolios_for_period" or "date_filtered_portfolios"